app/src/main/res/values-ar/strings.xml

```diff
@@ -56,8 +56,22 @@
     <string name="mark_as_read">ضع إشارة مقروء</string>
     <string name="mark_as_unread">وضع علامة كغير مقروءة</string>
     <string name="me">Me</string>
+    <!-- Archive -->
+    <string name="unarchive">Unarchive</string>
+    <string name="empty_archive">Remove all archived conversations</string>
+    <string name="skip_archive">Skip the archive, delete conversations directly</string>
```

Comment:
for the new way of archiving, lets create a separate menu button for Archive conversation. So this skipping the archive wont be needed anymore, it will be replaced by skipping the recycle bin.

Reply:
I forgot to remove unused strings, my bad. It is a separate menu option now and I have also removed the setting to enable it, since it is separate.